@xen-orchestra/web/src/components/host/HostPifSidePanel.vue

```diff
@@ -0,0 +1,459 @@
+                  v-tooltip="$t('coming-soon')"
+                  disabled
+                  :icon="faEllipsis"
+                  size="medium"
+                  accent="brand"
+                />
+              </template>
+            </VtsCardRowKeyValue>
+          </div>
+          <VtsCardRowKeyValue v-else>
+            <template #key>
+              {{ $t('ip-addresses') }}
+            </template>
+            <template #value>
+              <span class="value" />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- MAC ADDRESSES -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('mac-address') }}
+            </template>
+            <template #value>
+              {{ pif.mac }}
+            </template>
+            <template v-if="pif.mac" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(pif.mac)"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- NETMASK -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('netmask') }}
+            </template>
+            <template #value>
+              <span class="value">{{ pif.netmask }}</span>
+            </template>
+            <template v-if="pif.netmask" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(String(pif.netmask))"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- DNS -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('dns') }}
+            </template>
+            <template #value>
+              <span class="value">
+                {{ pif.dns }}
+              </span>
+            </template>
+            <template v-if="pif.dns" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(String(pif.dns))"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- GATEWAY -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('gateway') }}
+            </template>
+            <template #value>
+              <span class="value">
+                {{ pif.gateway }}
+              </span>
+            </template>
+            <template v-if="pif.gateway" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(String(pif.gateway))"
```

Comment:
I will not add comments on each section, but please refer to my previous comments about the `v-if` and `String()` parts